package.json

```diff
@@ -35,7 +41,9 @@
     "clean": "lerna run clean",
     "lint": "lerna run lint",
     "lint:fix": "lerna run lint:fix",
-    "custom-version": "yarn lerna version --conventional-commits=false"
+    "custom-version": "yarn lerna version --conventional-commits=false",
+    "export-grpc-stubs": "protoc --proto_path=./common/protocol/src/types --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_opt=snakeToCamel=false --ts_proto_opt=outputServices=grpc-js --ts_proto_out=./common/docker/src common/protocol/src/types/protos/runtime.proto"
```

Comment:
@kchrist-rocketfueldev I think this should be integrated in a `scripts` directory inside `common`. We had discussed that devs would run the script to generate new gRPC stubs if the `.proto` file has changed before starting/resuming their work.

This should be able to work for any language but right now make it only for TS until we develop the other runtimes.

Reply:
Also the `export-grpc-stubs` script that does not reference the correct path of the `runtime.proto`.

Reply:
added a separate script for export grpc stubs for TS. The script will be updated in the future to support other languages